python/paddle/fluid/executor.py

```diff
@@ -225,7 +226,19 @@ def run(self,
             feed_var_name='feed',
             fetch_var_name='fetch',
             scope=None,
-            return_numpy=True):
+            return_numpy=True,
+            use_program_cache=False):
+        """
+        :param program: the program that need to run
+        :param feed: feed variable list
+        :param fetch_list: fetch variable list
+        :param feed_var_name: feed_var_name default to 'feed'
```

Comment:
Maybe a bit more comment here? I'm confused about the difference between "fetch_list" and "fetch_var_name". And the meaning of default 'feed'/'fetch'.

Reply:
done